algoliasearch/build.gradle

```diff
@@ -21,7 +21,17 @@
  * THE SOFTWARE.
  */
 
+buildscript {
+    repositories {
+        jcenter()
```

Comment:
I would put here `mavenCentral()` - it is originally deployed to Maven Central and by the way mirrored to jcenter.